scripts/AddCopyrightHeaders.ps1

```diff
@@ -25,7 +21,7 @@ function Add-CopyrightHeaders($basePath)
 		if ($fileContent.StartsWith($copyrightHeaderString) -eq $false)
 		{
 			# Add the copyright header to the file
-			Set-Content $sourceFile.FullName ($copyrightHeaderString + "`r`n`r`n" + $fileContent)
+			Set-Content $sourceFile.FullName ($copyrightHeaderString + [Environment]::NewLine + [Environment]::NewLine + $fileContent)
```

Comment:
Slightly fixed up this script, but it also removes the BOM and adds an empty line to the end of files. We should replace this with existing tooling if available.